juniper_subscriptions/src/lib.rs

```diff
@@ -19,6 +19,11 @@ use juniper::{
     BoxFuture, ExecutionError, GraphQLError, GraphQLSubscriptionType, GraphQLTypeAsync, Object,
     ScalarValue, SubscriptionConnection, SubscriptionCoordinator, Value, ValuesStream,
 };
+#[cfg(feature = "unstable-ws-subscriptions-transport")]
+/// Utilities for the implementation of subscriptions over websocket
+pub mod ws_util;
```

Comment:
I'd like this to be more descriptive / align more with the proposed spec naming.